### PR TITLE
ossia.score version 3.1.4

### PR DESCRIPTION
**Pull request by dongle**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/89315)

### CODE REVIEW

**Comment by dongle** [comment]
@microsoft-github-policy-service agree

File: manifests/o/ossia/score/3.1.4/ossia.score.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ossia.score
+PackageVersion: 3.1.4
+PackageLocale: en-US
+Publisher: ossia
+PackageName: score
```
Comment (stephengillie) [inline manifests/o/ossia/score/3.1.4/ossia.score.locale.en-US.yaml:8]:
```suggestion
PackageURL: https://ossia.io/
PackageName: score
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Please see https://github.com/microsoft/winget-pkgs/pull/89315#discussion_r1029755501

File: manifests/o/ossia/score/3.1.4/ossia.score.locale.en-US.yaml (hunk 1 above)
Comment (dongle) [reply manifests/o/ossia/score/3.1.4/ossia.score.locale.en-US.yaml:8]: Thank you for your guidance on this and my other submissions!

File: manifests/o/ossia/score/3.1.4/ossia.score.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ossia.score
+PackageVersion: 3.1.4
+PackageLocale: en-US
+Publisher: ossia
+PackageURL: https://ossia.io/
```
Comment (stephengillie) [inline manifests/o/ossia/score/3.1.4/ossia.score.locale.en-US.yaml:8]:
```suggestion
PackageUrl: https://ossia.io/
```

Comment (stephengillie) [reply manifests/o/ossia/score/3.1.4/ossia.score.locale.en-US.yaml:8]: Sorry, I had incorrect capitalization in the previous suggestion. I've tested this suggestion to be sure it's correct.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**